pages/services/kafka/2.6.0-2.2.1/started/index.md

```diff
@@ -0,0 +1,617 @@
+
+* `ROOT`: Broker data is stored on the same volume as the agent work directory. Broker tasks will use the configured amount of disk space.
+* `MOUNT`: Broker data will be stored on a dedicated volume attached to the agent. Dedicated MOUNT volumes have performance advantages and a disk error on these MOUNT volumes will be correctly reported to Kafka.
+
+Configure Kafka service to use dedicated disk volumes:
+* **DC/OS cli options.json**:
+
+```json
+    {
+        "brokers": {
+            "disk_type": "MOUNT"
+        }
+    }
+```
+
+* **DC/OS web interface**: Set the environment variable `DISK_TYPE`: `MOUNT`
+
+When configured to `MOUNT` disk type, the scheduler selects a disk on an agent whose capacity is equal to or greater than the configured `disk` value.
+
+## JVM Heap Size
+
+Kafka service allows configuration of JVM Heap Size for the broker JVM process. Use the following configuration options:
+
+```json
+    {
+        "brokers": {
+            "heap": {
+                "size": 2000
+            }
+        }
+    }
+```
+
+* **DC/OS web interface**: Set the environment variable `BROKER_HEAP_MB`: `2000`
+
+**Note**: The total memory allocated for the Mesos task is specified by the `BROKER_MEM` configuration parameter. The value for `BROKER_HEAP_MB` should not be greater than `BROKER_MEM` value. Also, if `BROKER_MEM` is greater than `BROKER_HEAP_MB` then the Linux operating system will use `BROKER_MEM` - `BROKER_HEAP_MB` for [PageCache](https://en.wikipedia.org/wiki/Page_cache).
+
+## Alternate ZooKeeper
+
+By default the Kafka framework uses the ZooKeeper ensemble made available on the Mesos masters of a DC/OS cluster. You can configure an alternate ZooKeeper at install time.
+
+Use the following configuration options:
+
+* **DC/OS CLI options.json**:
+
+```json
+    {
+        "kafka": {
+            "kafka_zookeeper_uri": "zookeeper.marathon.autoip.dcos.thisdcos.directory:2181"
+        }
+    }
+```
+
+This configuration option cannot be changed after installation.
+
+## Recovery and Health Checks
+
+You can enable automated replacement of brokers and configure the circumstances under which they are replaced.
+
+### Enable Broker Replacement
+
+To enable automated replacement using the following options:
+
+* **DC/OS CLI options.json**:
+
+```json
+    {
+        "enable_replacement":{
+            "description":"Enable automated replacement of Brokers. WARNING: May cause data loss. See documentation.",
+            "type":"boolean",
+            "default":false
+        }
+    }
+```
+
+* **DC/OS web interface**: Set the environment variable `ENABLE_REPLACEMENT`: `true` to enable replacement.
+
+<p class="message--warning"><strong>WARNING: </strong>text</p>
+**Warning:** The replacement mechanism is not aware of whether the broker has been destructively replaced with the latest copy of data. You may lose data depending on your replication policy, the degree, and duration of the permanent failures.
```

Comment:
Use the `Warning` message as listed in https://wiki.mesosphere.com/display/DOCS/Documentation+Style+Guide#DocumentationStyleGuide-Notes